src/business.ts

```diff
@@ -114,4 +129,54 @@ export class BusinessApi {
     const success = !resp?.payload?.errorCode
     return { success, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take the attributes of a new Business account, create that
+   * in Marqeta, and return the account information. If no attributes are
+   * provided, a new Business account will still be created and its
+   * token ID returned.
+   */
+  async create(create: {
+     business: Business
+  }): Promise<{
+    success: boolean,
+    body?: Business,
+    error?: MarqetaError,
+  }> {
+    const searchOptions = snakecaseKeys(create?.business)
+    const resp = await this.client.fire(
+      'POST',
+      'businesses',
+      {},
+      { ...searchOptions }
+    )
+    // catch 404s...
+    if (resp?.payload?.errorCode >='404000' && resp?.payload?.errorCode <= 404999) {
+      const error = resp?.payload?.errorMessage
+      return {
+        success: false,
+        error: {
+          type: 'marqeta',
+          error,
+          status: resp?.payload?.errorCode,
+          marqetaStatus: resp?.payload?.errorCode
+        },
+      }
+    }
+    // catch all other errors...
+    if (resp?.payload?.errorCode > 404999){
+      const error = resp?.payload?.errorMessage
+      return {
+        success: false,
+        error: {
+          type: 'marqeta',
+          error,
+          status: resp?.payload?.errorCode,
+          marqetaStatus: resp?.payload?.errorCode
+        },
+      }
+    }
+    const success = !resp?.payload?.errorCode
+    return { success, body: { ...resp.payload } }
```

Comment:
Two things... first, if the evaluation of `success` is this simple, make it one line:

```typescript
    return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
```

second... I think there's got to be some "successful" code on the response - other than the _absence_ of an `errorCode` - Right? Isn't there a "successful" _something_ in the response?

For the Business, you could even just do:

```typescript
    return { success: !resp?.payload?.status, body: { ...resp.payload } }
```

or:

```typescript
    return { success: !resp?.payload?.token, body: { ...resp.payload } }
```

both are supposed to be in the valid output - Right?

Reply:
There is no successful code in the response upon Business account creation.  The response is the Business json.  If no token exists and the response contains errorCode and errorMessage, then no Business was created. 

response success:
https://www.marqeta.com/docs/core-api/businesses#_sample_response_body

response error:
`{"errorMessage":"Resource not found or unavailable","errorCode":"404000"}`